assets/js/modules/tagmanager/components/common/AMPContainerSelect.test.js

```diff
@@ -50,6 +50,8 @@ describe( 'AMPContainerSelect', () => {
 		registry.dispatch( STORE_NAME ).setAccountID( accountID );
 		registry.dispatch( STORE_NAME ).receiveGetAccounts( [ account ] );
 		registry.dispatch( STORE_NAME ).receiveGetContainers( [ ...webContainers, ...ampContainers ], { accountID } );
+		registry.dispatch( STORE_NAME ).finishResolution( 'getAccounts', [] );
+		registry.dispatch( STORE_NAME ).finishResolution( 'getContainers', [ accountID ] );
```

Comment:
Maybe we can group each `finishResolution` call with the corresponding `receiveGet...` call? That would make the connection more clear when reading the code.

Reply:
Yep, makes sense. Updated.